README.md

```diff
@@ -77,7 +77,11 @@ Additional fields are NOT allowed for any object type.
         "isDerived": false,
         "properties": [ ... ],
         "literals": null,
-        "propertyAssignments": { ... }
+        "propertyAssignments": { ... },
+        "isExtensional": null,
+        "isPowertype": null,
+        "order": null,
```

Comment:
Why don't you set order to "1" here is this example? This is the default case, right?

Reply:
Am I correct to conclude that the minimum value for `order` is 2? If so, shouldn't we add this to the field description?

Reply:
I have just seen your following comment about the minimum value.

Here is my proposal for the new description:

`"A string field that captures the type-order of a class decorated as «type». For ordered types, its value MUST be an integer greater than 2. For orderless types, its value MUST be \"*\"."`

Reply:
I have just seen your last comment! 😂😂
I really should read everything before commenting... 

So I guess you concluded that it is preferable to have a consistent use of the `"type"` field for all class elements.

`"A string field that captures the type-order of a class. In classes decorated with «type», this field MUST either be: a string containing an integer greater or equal than 2, if the type is ordered; or \"*\" if the type is orderless. For classes decorated with any other stereotype, this field MUST be set to \"1\"."`